docs/en/sql-reference/table-functions/generate_series.md

```diff
@@ -0,0 +1,25 @@
+---
+slug: /en/sql-reference/table-functions/generate_series
+sidebar_position: 146
+sidebar_label: generate_series
+---
+
+# generate_series
```

Comment:
For consistency with `generateRandom`, this should really be called `generateSeries`.

Reply:
I think we shoud add `alias` for both versions.